doc/source/async_api.rst

```diff
@@ -110,3 +133,32 @@ You can limit the number of concurrent task running at once using the
     # Only 10 tasks will be running concurrently. Once 10 finish, the next 10 should run.
     ray.get([actor.run_task.remote() for _ in range(50)])
 
+.. _threaded-actors:
+
+Threaded Actors
+---------------
+
+Sometimes, asyncio is not an ideal solution for your actor. For example, you may
+have one method that performs some computation heavy task while blocking the event loop, not giving up control via ``await``. This would hurt the performance of an Async Actor because Async Actors can only execute 1 task at a time and rely on ``await`` to enable multiplexing.
```

Comment:
```suggestion
have one method that performs some computation heavy task while blocking the event loop, not giving up control via ``await``. This would hurt the performance of an Async Actor because Async Actors can only execute 1 task at a time and rely on ``await`` to context switch.
```